Example/README.md

```diff
@@ -0,0 +1,78 @@
+# Example App
+
+This guide walks you through the Example app implementation showcasing how 100ms SDK can be utilized to build delightful apps.🤩
+
+You can download the [Sample App via TestFlight here](https://testflight.apple.com/join/dhUSE7N8).
```

Comment:
We have not been pushing updates to testflight for quite some time. I think we should add this line when we actually have an app on appstore that is maintained. Lets remove for now.

Reply:
Sure

Reply:
@gzerad Made the change, removed Testflight links